docs/feature_macros.md

```diff
@@ -193,6 +193,8 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 };
 ```
 
+?> Enumerated list of custom keycodes (`enum custom_keycodes`) must always be declared before `keymaps[]` array, `process_record_user()` and any other function that uses them for the compiler to recognise the list.
```

Comment:
I feel like this implies that the enum must be added multiple times before each use case. Maybe something like this would be better? Certainly open to other options.
```suggestion
?> The enumerated list of custom keycodes `enum custom_keycodes` must be declared only once and before the `keymaps[]` array or any other function that uses custom keycodes.
```

Reply:
Updated sentence grammar to reference a single list to avoid ambiguity.